ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -52,6 +52,14 @@
   : $doc.display($propName)
   #end
 #end
+#macro (setTitle)
```

Comment:
You should be more specific here and name the macro something like `setPollTitle` and also rename accordingly the `$title` variable defined at the end of the macro.